src/bots/slack/conversations/message.conversation.ts

```diff
@@ -0,0 +1,11 @@
+import { SlackBotWorker } from 'botbuilder-adapter-slack';
```

Comment:
I think we should finalize the filing for this during the meeting. `conversations` is helpful, but excludes other slack features we may add with botkit, like slash_commands and files for interactive messages. 

We'll need to think about how we want to organize large conversations and related files.

Reply:
ok, yes let's